lib/Test.pm6

```diff
@@ -489,7 +489,7 @@ multi sub like(
     $time_after = nqp::time_n;
     $got.defined; # Hack to deal with Failures
     my $ok;
-    if $got ~~ Str:D {
+    if Str($got) ~~ Str:D {
```

Comment:
Here, you're coercing to a Str and then checking if the result is a Str.... It's extremely unlikely the result would be false, so why keep the `else` branch around?